src/lib/secrets/wallets.ml

```diff
@@ -26,21 +43,17 @@ let load ~logger ~disk_location : t Deferred.t =
   let%bind files = Sys.readdir path >>| Array.to_list in
   let%bind keypairs =
     Deferred.List.filter_map files ~f:(fun file ->
-        if Filename.check_suffix file ".pub" then return None
-        else
-          match%map
-            Secret_keypair.read ~privkey_path:(path ^/ file) ~password
-          with
-          | Ok keypair ->
-              Some (keypair.public_key |> Public_key.compress, keypair)
-          | Error e ->
-              Logger.error logger ~module_:__MODULE__ ~location:__LOC__
-                "Error reading key pair at $path/$file: $error"
-                ~metadata:
-                  [ ("file", `String file)
-                  ; ("path", `String path)
-                  ; ("error", `String (Error.to_string_hum e)) ] ;
-              None )
+        match String.chop_suffix file ~suffix:".pub" with
+        | Some file_base -> (
```

Comment:
nit: shouldn't this be file_name? It seems clearer for me

Reply:
renamed to `sk_filename`